### ISSUE TITLE
Image carousel not listening to keyboard events and aspect ratio off

### ISSUE THREAD

**Issue by KevinTriplett** [Hylozoic/hylo-evo#1667]
When an image is clicked from a PostCard, the keyboard events are not connecting to the listener, only after the PostDetails are showing on the right-hand side and the PostCard image is clicked are the listeners connected.

So it must be kind of like a race condition.

**Comment by KevinTriplett** [comment]
Also, adding this here: I posted three memes with the same aspect ratio and the two thumbnails in the post details view were squashed.

**Comment by KevinTriplett** [comment]
Opening new issue, this one is invalid
